src/paymentdisclosuredb.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2017 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef ZCASH_PAYMENTDISCLOSUREDB_H
+#define ZCASH_PAYMENTDISCLOSUREDB_H
+
+#include "paymentdisclosure.h"
+
+#include <cstdint>
+#include <string>
+#include <mutex>
+#include <future>
+#include <memory>
+
+#include <boost/optional.hpp>
+
+#include <leveldb/db.h>
```

Comment:
Non-blocking: we should migrate to `dbwrapper.h` once #2598 is merged.

Reply:
Ok